engine/tests/settings.cpp

```diff
@@ -52,12 +52,21 @@ TEST_CASE("cubos::engine::settingsPlugin")
     };
 
     EmbeddedArchive::registerData("settings.json", settingsData);
-    FileSystem::mount("/settings.json", std::make_unique<EmbeddedArchive>("settings.json"));
+
+    if (!FileSystem::mount("/settings.json", std::make_unique<EmbeddedArchive>("settings.json")))
+    {
+        FileSystem::unmount("/settings.json");
+        FileSystem::mount("/settings.json", std::make_unique<EmbeddedArchive>("settings.json"));
+    }
```

Comment:
? what is going on here

Reply:
This test was failing because "/settings.json" was already mounted from executing the raycast test, getting a "could not mount archive error". I don't really know how the file system works so I just unmounted and remounted.